### PR TITLE
Reword security and privacy section

### PR DESCRIPTION
**Pull request by alexshalamov**
This PR adds links to research papers and possible mitigation strategies for inertial sensors.

***
<a href="https://pr-preview.s3.amazonaws.com/alexshalamov/accelerometer/pull/38.html" title="Last updated on Feb 28, 2018, 3:06 PM GMT (03a00c0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/accelerometer/38/b50b90a...alexshalamov:03a00c0.html" title="Last updated on Feb 28, 2018, 3:06 PM GMT (03a00c0)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -119,8 +162,18 @@ Sensor use cases</a></cite> documents.
 Security and Privacy Considerations {#security-and-privacy}
 ===================================
 
-There are no specific security and privacy considerations
-beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
+Sensor readings provided by inertial sensors, such as accelerometer, could be used by adversaries
```
Comment (pozdnyakov) [inline index.bs]: [=Sensor readings=]

Comment (anssiko) [inline index.bs]: Rewording suggestion:
```
Security threats related to [=sensor readings=] provided by inertial sensors such as accelerometer include,
for example, [=keylogging=], [=location tracking=], [=fingerprinting=] and [=user identifying=].
```

File: index.bs [hunk 2]
```diff
@@ -119,8 +162,18 @@ Sensor use cases</a></cite> documents.
...
+to exploit various security threats, for example, [=keylogging=], [=location tracking=],
+[=fingerprinting=] and [=user identifying=].
+
+Research papers published by security community, for instance, [[KEYSTROKEDEFENSE]], indicate that
+by throttling the frequency, risks of successful attacks is not fully eliminated, while may greatly
```
Comment (anssiko) [inline index.bs]: `by throttling the frequency, risks of successful attacks are not fully eliminated, while throttling may greatly`

File: index.bs [hunk 3]
```diff
@@ -119,8 +162,18 @@ Sensor use cases</a></cite> documents.
...
+affect usefulness of a web application with ligitimate reasons to use the sensors.
+
+The [[TOUCHSIGNATURES]] and [[ACCESSORY]] research papers propose that the user agent may
```
Comment (anssiko) [inline index.bs]: implementations can

(In informative text, better use "implemetation" and talk about "user agent" in normative parts. Also "may" is RFC 2119 term and better use "can" instead in informative text.)

**Pull request merged**